db-server/index.js

```diff
@@ -97,6 +97,7 @@ function createServer(db) {
       return db.deleteEmail(req.params.id, req.params.email)
     })
   )
+
```

Comment:
Interesting, looks like this might have been missed on the original secondary email implementation. Opened issue https://github.com/mozilla/fxa-auth-db-mysql/issues/311.